### ISSUE TITLE
Installed this over docker, but api calls seems to be not working

### ISSUE THREAD

**Issue by anantsparrow** [XposedOrNot/XposedOrNot-API#11]
**Describe the bug**
(http://5.78.109.84:8000/v1/check-email/test@example.com)
this is the url where i have installed the docker and installed your project , but the api is giving me not found value everytime, instead of a response, so can you help with this?

**Comment by DevaOnBreaches** [comment]
Hello @anantsparrow,

Thank you for raising this question.

Please ensure you have enabled [Google Datastore](https://cloud.google.com/datastore/) and authenticated either directly online or in the local emulator, as outlined in the [installation requirements](https://github.com/XposedOrNot/XposedOrNot-API?tab=readme-ov-file#quick-start-for-local-development).

Additionally, the search criteria will need data to display results. For instance, if your Datastore is not populated with sample data, you might not see any results.

Please let me know if you are trying to use this as a service or if you are developing/adding more features to it.

If you are just looking to use it, the direct API routes and website will be helpful. If you are looking to develop/add more features to this API, you may need to start populating some content into the primary tables like xon, xon_alert, etc., to see output.

**Comment by anantsparrow** [comment]
Let me try again, i am not that much technical in this, i will update you if i face any error
